OPAL/framework/src/main/resources/ApplicationProject.conf

```diff
@@ -0,0 +1,21 @@
+org.opalj.br.analyses {
```

Comment:
This already exists identically as `CommandLineProject.conf` in the same directory. Why was this added?